.circleci/config.yml

```diff
@@ -123,44 +117,49 @@ jobs:
               - content
   validate-files:
       docker:
-        - image: cimg/python:3.9.10-node
+        - image: << pipeline.parameters.global-docker-image >>
       steps:
         - checkout
         - attach_workspace:
             at: ~/project
         - <<: *install_demisto_sdk
+        - python/install-packages:
+            pkg-manager: "pip"
+            pip-dependency-file: "requirements-gsutil.txt"
         - run:
             name: Set Up & Create Id Set
             when: always
+            environment:
+              CI_COMMIT_BRANCH: "master"
```

Comment:
I think the value should be set by a pipeline parameter where the default value is `master`.

Reply:
It is wrong doing so. Run the SDK but the branch., We are running "master" checks on non-master branches,

Reply:
I don't follow.